core-bundle/src/Resources/contao/elements/ContentImage.php

```diff
@@ -36,6 +40,8 @@ class ContentImage extends ContentElement
 	 */
 	public function generate()
 	{
+		// Note: These file operations are only here for BC reasons and do not
+		//       serve any functional purpose anymore.
```

Comment:
I’m a little bit confused. Does’t this method set up `$this->objFilesModel` which is used afterwards in line 68 by `$this->getFigureBuilderIfResourceExists($this->objFilesModel);`?

Reply:
Ah yes, this comment is misleading. I initially had directly used `$this->singleSRC` as the resource for the `FigureBuilder` which made this whole method obsolete. But `singleSRC` being a path instead of a UUID and `objFilesModel` being actually set could be something people are relying on and I'm not sure we can drop for BC reasons. Wdyt? 

If we cannot drop it, we should imo just remove the comment.

Reply:
I think we should just remove the comment.

Reply:
:+1: removed in 11a205ea8b839433445719f1f1e200de8da21ced